### ISSUE TITLE
About  the problem of task_id 

### ISSUE THREAD

**Issue by dyl0101** [imatge-upc/liverseg-2017-nipsws#24]
Hi，I'm very interested in this work, and just started to do your work.I don't understand some of the code，so I would like to ask you a question：
Why is there only task_id=2 in seg_liver.py and seg_lesion.py? Didn't  see what to do when task_id = 1?
`if task_id == 2:
                    batch_label = batch_label_liver
                    batch_label_val = batch_label_liver_val
                label = preprocess_labels(batch_label, number_slices)
                label_val = preprocess_labels(batch_label_val, number_slices)
                run_res = sess.run([total_loss, merged_summary_op, dice_coef_op] + grad_accumulator_ops,feed_dict={input_image: image, input_label: label})
                batch_loss = run_res[0]
                summary = run_res[1]
                train_dice_coef = run_res[2]
`
